_rules/text-node-contrast-afw4f7.md

```diff
@@ -0,0 +1,337 @@
+---
+id: afw4f7
+name: Text node has minimal contrast
+rule_type: atomic
+
+description: |
+  This rule checks that every text node has minimal color contrast with its background
+
+accessibility_requirements:
+  wcag20:1.4.3: # Contrast (Minimum)
```

Comment:
may be it is representation in github, but these look like `tabs` over `spaces`

Reply:
Not seeing it. Resolve